README.rst

```diff
@@ -61,6 +62,7 @@ On Ubuntu-based Linux distributions the packages can be installed with *apt-get*
    apt-get install python3-pip
    apt-get install python-dbus
    apt-get install postgresql
+   apt-get install autoconf automake build-essential libtool python-dev
```

Comment:
I take it these are needed to build the pyjq library and not for runtime? This makes.a difference for snap production.

Reply:
Right,not required for runtime. I will get rid of them

Reply:
We need to know about build requirements as well, so this is good to have. I will speak to Ivan about how to deal with requirements like this.